packages/core/upload/admin/src/pages/App/MediaLibrary.js

```diff
@@ -339,13 +340,13 @@ export const MediaLibrary = () => {
                           id={`folder-${folder.id}`}
                           to={url}
                           startAction={
-                            selectOne && (
+                            selectOne && folder.isSelectable ? (
```

Comment:
`Invalid prop `startAction` of type `boolean` supplied to `ForwardRef`, expected a single ReactElement`

`should show breadcrumbs navigation` test failed with this error, not sure why I had to fix it this way only for this one test